spykes/django-admin-school/src/user/models.py

```diff
@@ -58,9 +71,31 @@ def __str__(self):
 
 
 class Employee(AppUser):
-    extra = models.CharField(
-        verbose_name=_('extra'),
-        max_length=40
+    #vacation = models.ForeignKey(
+    #    Vacation,
+    #    on_delete=models.DO_NOTHING,
+    #    verbose_name=_("vacation"),
+    #    null=True,
+    #    blank=True,
+    #)
+
+    vacation_days = models.IntegerField(
+        _("vacation days"),
+        default=0,
+        help_text=_("Number of vacation days available"),
+        blank=True
+    )
+
+    bank_account = models.CharField(
+        _("bank account"),
+        max_length=34,

```

Comment:
I would change max_length to a higher value as it is going to store the encrypted bacnk account number, also add the encryption functionality to the related signal (signals.py)